frontend/src/components/design-system/containers/CardComponent.vue

```diff
@@ -22,10 +22,8 @@ const props = withDefaults(defineProps<Props>(), {
   backgroundColor: 'rgb(var(--color-neutral-bg-container))',
   border: false,
   borderKind: 'normal',
-  // eslint-disable-next-line vue/require-valid-default-prop
   borderRadius: 'small',
-  // eslint-disable-next-line vue/require-valid-default-prop
-  padding: 'medium',
+  padding: Object('medium') as Props['padding'],
```

Comment:
without Object it throws bunch of errors in the console. I don't understand the reasons really